torch/nn/utils/parametrize.py

```diff
@@ -282,6 +283,14 @@ def _inject_new_class(module: Module) -> None:
     """
     cls = module.__class__
 
+    def default_deepcopy(self, memo=None):
+        # Just emulate a standard deepcopy procedure when __deepcopy__ doesn't exist in the current class.
+        memo = {} if memo is None else memo
+        memo[id(self)] = replica = self.__new__(self.__class__)
```

Comment:
Could you split this into two assignments for clarity?

Reply:
Sure! Done :heavy_check_mark: